app/controllers/sessions_controller.rb

```diff
@@ -26,15 +20,18 @@ def destroy
 
   private
 
-  def load_user
+  def load_user_and_check_login
```

Comment:
Sao phải sửa code cũ
mình làm từng step để có mess lỗi cụ thể và hạn chế dùng toán tử an toàn đi cu